src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/PartialUpdateSpec.groovy

```diff
@@ -333,9 +332,6 @@ class PartialUpdateSpec extends HealthCheckSpecification {
         Wrappers.wait(RULES_DELETION_TIME) {
             assert switchHelper.validateAndCollectFoundDiscrepancies([dstSwitch, newDstSwitch]*.dpId).isEmpty()
         }
-
-        cleanup:
-        switchHelper.synchronizeAndCollectFixedDiscrepancies([dstSwitch, newDstSwitch]*.dpId)
```

Comment:
I wasn't able to find the place where we added this action during test execution. It seems like a precautious cleanup step.

Reply:
It was exhaustive action because we executed the same synchronization two lines above. Also we don't do in test anything that could cause rules discrepancies. Must be an artifact from one of the previous refactorings.